src/jdk.compiler/share/classes/com/sun/tools/javac/tree/TreeInfo.java

```diff
@@ -848,10 +844,8 @@ public static JCTree skipParens(JCTree tree) {
 
     /** Skip parens and return the enclosed expression
      */
+    //XXX: remove??
```

Comment:
What about this?

Reply:
Should be resolved by:
https://github.com/openjdk/jdk/pull/13074/commits/bb59254e409821c9b14287c73a35ebe2dd2cbda9

Thanks for the comment!